CONTRIBUTING.md

```diff
@@ -1,6 +1,6 @@
 # Contributing
 
-- Your favourite command isn't covered?
+- Your favorite command isn't covered?
```

Comment:
> isn't a strong argument, because I have the opposite happening :)

That "argument" was precisely an invitation to get the perspective from the opposite side, i.e. whether there were several words highlighting US spellings in this document, even in the version before this PR. If so, that'd be evidence that US spelling was already prevalent on this doc so it would make sense standardizing on it -- otherwise, we could standardize in UK spelling instead (I just couldn't easily make sure I did that right, without taking extra steps).

But that's actually not even something we should be discussing in this PR -- I shouldn't have bundled unrelated changes together anyway. I'll revert the spelling changes with my next commit.